data/mods/ssb/moves.ts

```diff
@@ -1082,6 +1082,37 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		target: "normal",
 		type: "Ice",
 	},
+	// Celine
+	statusguard: {
+		accuracy: 100,
+		basePower: 0,
+		category: "Status",
+		desc: "Protects from physical moves. If hit by physical move, opponent is either badly poisoned, burned, or paralyzed at random and is forced out. Special attacks and status moves go through this protect.",
+		shortDesc: "If opponent uses physical attack, then gets status and phazed. User is protected.",
```

Comment:
This shortDesc is too long. A good way to test is by doing `/dt tackle` and clicking inspect on the description of the move and replacing it with your description. If it gets cut off, it's too long

Reply:
also it's phrased kinda weird

Reply:
Protects from damaging attacks. Contact: burn, toxic, or para.